TECH_DEBT_TRACKER.md

```
@@ -174,21 +167,22 @@ Consider splitting for maintainability:
+- `ARTIST_CAROUSEL.md`, `ARTIST_PROFILE_ENHANCEMENTS.md`, `ARTIST_PROFILE_UPDATES.md`
+- `ARTIST_IMAGE_MAINTENANCE.md`, `AUTH_MIGRATION_SUMMARY.md`, `DASHBOARD_LAYOUT_PATTERNS.md`
+- `DROPDOWN_MIGRATION_SUMMARY.md`, `HOMEPAGE_PERFORMANCE_TASKS.md`, `PERFORMANCE_ENHANCEMENTS.md`
+- `PERFORMANCE_ISSUES_AUDIT.md`, `SEO_ENHANCEMENTS.md`, `TEST_PERFORMANCE_OPTIMIZATION.md`
+- `PLAN-clerk-elements-migration.md`
 
 ---
 
 ## Change Log
 
 | Date | Author | Change |
 |------|--------|--------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)